runelite-client/src/main/java/net/runelite/client/plugins/idlenotifier/IdleNotifierPlugin.java

```diff
@@ -103,6 +104,7 @@ public class IdleNotifierPlugin extends Plugin
 	private Instant sixHourWarningTime;
 	private boolean ready;
 	private boolean lastInteractWasCombat;
+	private static final int buffBarNotDisplayed = -1;
```

Comment:
By convention, static final fields are always UPEPRCASE. Or just inline the -1 since it does not really matter.